_docs/_latest/configure/inventory/gsuite.md

```diff
@@ -70,3 +70,68 @@ If you are running Forseti on GCP and made any changes to the above values,
 you will need to copy the `conf` file to the Cloud Storage bucket. For more
 information, see
 [Moving configuration to Cloud Storage]({% link _docs/latest/configure/general/index.md %}).
+
+## Troubleshooting
+
+If have have problem with getting inventory on GSuite data, you can find what
+errors have happened by running `forseti inventory list|get`, or look at the
+`inventory_index_errors` column in the `inventory_index` table.
+
+Error:
+```
+('invalid_grant: Invalid email or User ID', u'{"error" : "invalid_grant", "error_description" : "Invalid email or User ID"}')
+```
+
+Solution:
+Double-check th email you entered in the `domain_super_admin_email` field of
+the `forseti_conf_server.yaml` file. Make sure there is no typo and the user
+exists.
+
+***
+
+Error:
+```
+GCP API Error: unable to get groups from GCP:
+<HttpError 403 when requesting https://www.googleapis.com/admin/directory/v1/groups?customer=C04h01n68&alt=json returned "Not Authorized to access this resource/api">
+```
+
+Solution:
+Make sure you specified a super admin user in the `domain_super_admin_email`
+field of the `forseti_conf_server.yaml` file.
+
+***
+
+Error:
+```
+('unauthorized_client: Client is unauthorized to retrieve access tokens using this method.', u'{"error" : "unauthorized_client", "error_description" : "Client is unauthorized to retrieve access tokens using this method."}')
+```
+
+Solution:
+Make sure you entered the correct API scope(s) in the GSuite admin console.
+
+***
+
+Error:
+```
+Null
```

Comment:
Just to make sure, is this the entire error message?

Reply:
Yes. Basically, there is no error reported in this case.  I am re-written the doc to be more clear.